soccer/gameplay/plays/restarts/placement.py

```diff
@@ -59,8 +59,8 @@ def on_enter_placing(self):
                                  required=True,
                                  priority=90)
 
-        #line_up = tactics.line_up.LineUp(self.create_lineup())
-        #self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
+        line_up = tactics.line_up.LineUp(self.create_lineup())
+        self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
```

Comment:
^ Yup agreed. We should either make a 'clearlineup' which lines up far away from the ball/target position, or make something more fancy. This is definitely a case we need to catch though, since placement will happen on the sidelines a lot. Thanks! 😄